cprover_bindings/src/irep/irep_id.rs

```diff
@@ -5,17 +5,18 @@
 //! This file contains [IrepId] which is the id's used in CBMC.
 //! c.f. CBMC source code [src/util/irep_ids.def]
 use crate::cbmc_string::InternedString;
-use num::bigint::BigInt;
+use crate::utils::NumUtils;
+use num::bigint::{BigInt, BigUint, Sign};
 
 #[derive(PartialEq, Eq, PartialOrd, Ord, Clone, Debug)]
 pub enum IrepId {
     /// In addition to the standard enums defined below, CBMC also allows ids to be strings.
     /// For e.g, to store the id of a variable. This enum variant captures those strings.
     FreeformString(InternedString),
-    /// An integer, encoded as a decimal string
+    /// An integer, to be encoded as a decimal string
     FreeformInteger(BigInt),
-    /// An integer, encoded as a hex string
-    FreeformHexInteger(BigInt),
+    /// A constant, stored as a bit pattern (negative numbers in 2's complement).
+    FreeformBitpattern(BigUint),
```

Comment:
Tiny little change. This should be Pascal case: FreeformBitPattern

Reply:
done